src/util.cpp

```diff
@@ -366,4 +366,24 @@ std::string get_nec_thread_comm(Thread thread)
     // If no '--' is found, fall back to the complete commandline as a name
     return std::accumulate(args.begin(), args.end(), std::string(""));
 }
+
+struct rlimit save_rlimit_fd()
```

Comment:
Don't call it `save`, if it only saves one time and then is used as getter. Also, which or whom rlimit is saved? Maybe `inherited_rlimit_fd`, `initial_rlimit_fd`, or `parent_process_rlimit_fd` instead?